magento/magento2
# Typo Mistake

### Description (*)


### Typo Mistake in core code.

1. app/code/Magento/Sales/Block/Adminhtml/Order/Creditmemo/Create/Adjustments.php

### Manual testing scenarios (*)

1. Open file from **app/code/Magento/Sales/Block/Adminhtml/Order/Creditmemo/Create/Adjustments.php** and check line number 63 and find `'code' => 'agjustments'` spelling is wrong.


### Contribution checklist (*)
